### PR TITLE
Add a " const " before "alg" and "typ" variable

### PR DESCRIPTION
**Pull request by Sardashti**
Hi and Thank you for your wonderful application

To not display the warning : 
" ISO C++ forbids converting a string constant to 'char*' " Add a "const" before "alg" and "typ" variable

Please apply my changes if approved.

### CODE REVIEW

File: src/CustomJWT.h [hunk 1]
```diff
@@ -157,7 +157,7 @@ class CustomJWT
               size_t outLen,
               char *alg,
```
Comment (Ant2000) [inline src/CustomJWT.h]: Any reason alg in this constructor was not made const?

File: src/CustomJWT.h [hunk 2]
```diff
@@ -118,7 +118,7 @@ class CustomJWT
               size_t maxSigLen, 
               char *alg,
```
Comment (Ant2000) [inline src/CustomJWT.h]: Here too?

File: src/CustomJWT.h (hunk 1 above)
Comment (Sardashti) [reply src/CustomJWT.h]: I'm so sorry, please check again. thank's

File: src/CustomJWT.h (hunk 2 above)
Comment (Sardashti) [reply src/CustomJWT.h]: I'm so sorry, please check again. thank's

**Pull request merged**